azure/defaults.go

```diff
@@ -331,7 +338,11 @@ func GetDefaultWindowsImage(k8sVersion, runtime string) (*infrav1.Image, error)
 		return nil, errors.New("containerd image only supported in 1.22+")
 	}
 
-	skuID, err := getDefaultImageSKUID(k8sVersion, "windows", "2019")
+	if osAndVersion == "" {
+		osAndVersion = fmt.Sprintf("windows-%s", DefaultWindowsServerVersion)
```

Comment:
nit: can we just make `windows-` part of the constant?

Reply:
I'll do the same for ubuntu

Reply:
updated (and squashed commits)